lib/theme.ts

```diff
@@ -10,7 +9,7 @@ declare module '@material-ui/core/styles/createTypography' {
     }
 }
 
-const THEME: ThemeOptions = {
+const THEME: DeprecatedThemeOptions = {
```

Comment:
Same as the other template, is it possible to make this the new `ThemeOptions` without too much headeache? Thank you.